### PR TITLE
Update the installation guide for the python version of Compose

### PR DESCRIPTION
**Pull request by glours**
**What I did**

Created a dedicated INSTALL.md file with the official documentation to install Python version as the official doc will refer to Compose `v2` which is going GA

**(not mandatory) A picture of a cute animal, if possible in relation with what you did**

image

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+> methods, you probably do not have the proper permissions to remove
+> `docker-compose`. To force the removal, prepend `sudo` to either of the above
+> commands and run again.
+
+## Using Compose v1 and Compose v2 on the same computer
+
+If you need to run both `v1` and `v2` versions of Compose you can use Compose Switch project that will allow you to 
+choose which version of compose to run.
+
+You can find all the installation and configuration step in the [project repository](https://github.com/docker/compose-switch)
```
Comment (ndeloof) [inline INSTALL.md:140]: do we really want to promote compose-switch? compose v2 can be used standalone, so v1 vs v2 is just a question on setting `PATH` to select the right one

Comment (glours) [reply INSTALL.md]: removed 👍

**Review by ndeloof — APPROVED** [review]

File: INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+This page contains information on how to install Docker Compose. You can run Compose on macOS, Windows, and 64-bit Linux.
+
+## Prerequisites
+
+Docker Compose relies on Docker Engine for any meaningful work, so make sure you
+have Docker Engine installed either locally or remote, depending on your setup.
+
+
+- first install the
```
Comment (usha-mandya) [inline INSTALL.md:11]:
```suggestion
- Install
```

File: INSTALL.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+  [Docker Engine](https://docs.docker.com/engine/install/#server)
+  for your OS as described on the Get Docker page, then come back here for
```
Comment (usha-mandya) [inline INSTALL.md:13]:
```suggestion
  for your OS and then refer to this page for
```

File: INSTALL.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+  instructions on installing Python version of Compose.
```
Comment (usha-mandya) [inline INSTALL.md:14]:
```suggestion
  instructions on installing the Python version of Compose.
```

File: INSTALL.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+
+- To run Compose as a non-root user, see [Manage Docker as a non-root user](https://docs.docker.com/engine/install/linux-postinstall/).
+
+## Install Compose
+
+Follow the instructions below to install Compose using the `pip`
+Python package manager or installing Compose as a container.
```
Comment (usha-mandya) [inline INSTALL.md:21]:
```suggestion
Python package manager or to install Compose as a container.
```

File: INSTALL.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+- [Install using pip](#install-using-pip)
+- [Install as a container](#install-as-a-container)
+
+#### Install using pip
+
+> For `alpine`, the following dependency packages are needed:
+> `py-pip`, `python3-dev`, `libffi-dev`, `openssl-dev`, `gcc`, `libc-dev`, `rust`, `cargo`, and `make`.
+{: .important}
+
+Compose can be installed from
```
Comment (usha-mandya) [inline INSTALL.md:43]:
```suggestion
You can install Compose from
```

File: INSTALL.md [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+
+```console
+$ sudo pip install docker-compose
+```
+
+> pip version 6.0 or greater is required.
+
+#### Install as a container
+
+Compose can also be run inside a container, from a small bash script wrapper. To
```
Comment (usha-mandya) [inline INSTALL.md:66]:
```suggestion
You can also run Compose inside a container, from a small bash script wrapper. To
```

File: INSTALL.md [hunk 8]
```diff
@@ -0,0 +1,133 @@
...
+install compose as a container run this command:
```
Comment (usha-mandya) [inline INSTALL.md:67]:
```suggestion
install Compose as a container run this command:
```

File: INSTALL.md [hunk 9]
```diff
@@ -0,0 +1,133 @@
...
+```
+
+Alternatively, if you're not worried about keeping them, you can remove them.
+Compose just creates new ones.
+
+```console
+$ docker container rm -f -v myapp_web_1 myapp_db_1 ...
+```
+
+## Uninstallation
```
Comment (usha-mandya) [inline INSTALL.md:114]:
```suggestion
## Uninstall
```

File: INSTALL.md [hunk 10]
```diff
@@ -0,0 +1,133 @@
+# Install Docker Compose
+
+This page contains information on how to install Docker Compose. You can run Compose on macOS, Windows, and 64-bit Linux.
```
Comment (thaJeztah) [inline INSTALL.md:3]: Perhaps add a ⚠️ and mention that this is about Compose V1, and that we recommend installing V2 (and link to docs.docker.com)

**Pull request merged**